### PR TITLE
update reference year to 2017

### PR DESCRIPTION
**Pull request by dwolfhub**

### CODE REVIEW

**Comment by stephtr** [comment]
Why don't replace `REFERENCE_YEAR` by `new Date().getFullYear()`?

**Comment by dwolfhub** [comment]
@stephtr good point. updated.

**Comment by lowe** [comment]
Thanks!

**Review by BGovertime — COMMENTED** [review]
Thank

**Review by Banshee3406 — APPROVED** [review]

File: src/scoring.coffee [hunk 1]
```diff
@@ -250,7 +250,7 @@ scoring =
     base_guesses * match.token.length
 
   MIN_YEAR_SPACE: 20
-  REFERENCE_YEAR: 2016
```
Comment (Banshee3406) [inline src/scoring.coffee]: 2030

**Pull request merged**